CRM/Core/Permission/WordPress.php

```diff
@@ -58,7 +58,20 @@ public function check($str) {
       return TRUE;
     }
 
-    require_once ABSPATH . WPINC . '/pluggable.php';
+	//CRM-15629
+	// During some extern/* calls we don't bootstrap CMS hence
+	// below constants are not set. In such cases, we don't need to
+	// check permission, hence directly return TRUE
+	if ( ! defined( 'ABSPATH' ) || ! defined( 'WPINC' ) ) {
```

Comment:
I left this section based on Kurund's orginal notes.   When the IPN comes back from PayPal, we don't need WP bootstrapped to complete the transaction.  However, emails are not sent as WP being bootstrapped is needed (line 68).  

I'll test without this.